torch_geometric/contrib/flag/flag.py

```diff
@@ -24,13 +58,39 @@ def __init__(
 
     def forward(
         self,
-        x: torch.tensor,
-        edge_index: torch.tensor,
-        y_true: torch.tensor,
-        train_idx: torch.tensor,
+        x: torch.Tensor,
+        edge_index: torch.Tensor,
+        y_true: torch.Tensor,
+        train_idx: torch.Tensor,
         step_size: float,
         n_ascent_steps: int = 3,
-    ) -> tuple[torch.nn.Module, torch.tensor]:
+    ) -> tuple[torch.nn.Module, torch.Tensor]:
+        """
+        Perform the FLAG forward and backward passes. Note that users should
+        not invoke loss.backward() or optimizer.step() externally to this
```

Comment:
Are we able to wrap `loss.backward()` and `optimizer.step()` in backticks to get inline code formatting in Sphinx?

Reply:
Should be able to! Will add